src/zenml/client.py

```diff
@@ -2564,6 +2565,12 @@ def _validate_stack_component_configuration(
                 "other users. You should consider using a non-local stack "
                 "component alternative instead."
             )
+        if not configuration.is_valid:
+            raise StackComponentValidationError(
+                f"Invalid stack component configuration or uninstalled integration. please verify "
+                f"the configurations set for {component_type} "
+                f"and Please ensure that the necessary integration is installed."
```

Comment:
If we return `True` in the import error case, this message would need to be adjusted again

Reply:
Yep, adjusted